libs/nuxt/src/builders/server/builder.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  BuilderContext,
+  BuilderOutput,
+  createBuilder,
+  targetFromTargetString,
+} from '@angular-devkit/architect';
+import { join, JsonObject, normalize } from '@angular-devkit/core';
+import { build, loadNuxt } from 'nuxt';
+import { concat, from, Observable } from 'rxjs';
+import { map, switchMap } from 'rxjs/operators';
+import { ServerBuilderSchema } from './schema';
+import { BrowserBuilderSchema } from '../browser/schema';
+import { getProjectRoot } from '../../utils';
+
+const serverBuilderOverriddenKeys = [];
+
+export function runBuilder(
+  options: ServerBuilderSchema,
+  context: BuilderContext
+): Observable<BuilderOutput> {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  async function setup(): Promise<any> {
+    const browserTarget = targetFromTargetString(options.browserTarget);
+    const rawBrowserOptions = await context.getTargetOptions(browserTarget);
+    const overrides = Object.keys(options)
+      .filter(
+        (key) =>
+          options[key] !== undefined &&
+          serverBuilderOverriddenKeys.includes(key)
+      )
+      .reduce((previous, key) => ({ ...previous, [key]: options[key] }), {});
+    const browserName = await context.getBuilderNameForTarget(browserTarget);
+    const browserOptions = await context.validateOptions<
+      JsonObject & BrowserBuilderSchema
+    >({ ...rawBrowserOptions, ...overrides }, browserName);
+
+    const projectRoot = await getProjectRoot(context);
+
+    const nuxt = await loadNuxt({
+      for: options.dev ? 'dev' : 'start',
```

Comment:
Running `nx serve my-app --dev false` works, but there is no output on the command line so it looks like it is hanging.

Reply:
The server URL is now logged.

<img width="565" alt="Screen Shot 2020-09-07 at 7 10 45 PM" src="https://user-images.githubusercontent.com/14145352/92421773-e080ea00-f13f-11ea-9a2b-8f63bcd798ab.png">

<img width="496" alt="Screen Shot 2020-09-07 at 7 11 23 PM" src="https://user-images.githubusercontent.com/14145352/92421781-e5459e00-f13f-11ea-9b87-d78edef6bced.png">